neonvm/controllers/virtualmachine_controller.go

```diff
@@ -598,6 +606,29 @@ func affinityForVirtualMachine(virtualmachine *vmv1.VirtualMachine) *corev1.Affi
 	return a
 }
 
+func notifyRunner(vm *vmv1.VirtualMachine, r resource.Quantity) error {
+	client := http.Client{Timeout: 5 * time.Second}
```

Comment:
does neonvm have contexts? feels like we'd want contexts here.

Reply:
It does, I added the code to pass it, thanks!